ee/ucp/release-notes.md

```diff
@@ -69,7 +69,7 @@ upgrade your installation to the latest release.
 
 ### New platforms
 * Added support for SLES 15.
-* Added support for Oracle 7.6.
```

Comment:
@ollypom as per @david-yu and https://success.docker.com/article/compatibility-matrix, Oracle 7.6 is still supported. Going to check on CentOS 7.6 with @andrewhsu...

Reply:
@bermudezmt Yep, but Oracle 7.6 was a "New Platform" in 3.1.3, mentioned further down in the release notes. Hence why I think adding it a New Platform in 3.1.4 was a typo, and instead it should of read CentOS :)

Reply:
Ah, great point. Still waiting on confirmation for CentOS 7.6 support. cc: @david-yu